src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -170,6 +225,24 @@ class EmojiPickerMenu extends Component {
                     getItemLayout={this.getItemLayout}
                     showsVerticalScrollIndicator
                 />
+                {Boolean(this.state.value) && (
```

Comment:
is there a benefit of using `Boolean` over `this.state.length > 0`?

Reply:
I prefer to use `!!this.state.value` but in case with `Boolean(this.state.value)` and `this.state.value.length > 0` the performance difference is likely to be negligible
`this.state.value.length > 0` can be faster because it avoids the additional conversion step.